app.rb

```diff
@@ -109,11 +109,13 @@ class InvalidChannelName < StandardError; end
   end
 
   get '/chef/install.msi' do
-    redirect to('/stable/chef/download?p=windows&pv=2008r2&m=x86_64')
+    # default to 32-bit architecture for now
+    redirect to('/stable/chef/download?p=windows&pv=2008r2&m=i686')
```

Comment:
@schisamo since the official omnitruck API doc used i686, I wanted to conform to that - https://docs.chef.io/api_omnitruck.html

I'm fine changing it either ways - please confirm,